test/unit/spec/EnrollChoices_spec.js

```diff
@@ -254,6 +254,11 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, Router,
             expect(test.form.submitButtonText()).toBe('Configure factor');
           });
         });
+        itp('does not have link to skip set up during grace period when all are required', function () {
```

Comment:
I don't think we need to say anything about `grace period` in the tests in general. At the end are showing the link only when it makes sense (one required factor enrolled and some required factors still not enrolled).
So this test can be `does not have skip setup link when all are required and none are enrolled`